### DISCUSSION TITLE
Will livewire update other browser windows automatically?

### DISCUSSION THREAD

**Discussion by lasseeee** [livewire/livewire#3681] [1. Help]
If i update some DB value through a Livewire component in one browser window, will that same value automatically be updated in other browser windows, or do I need to pool for changes to make the change appear in other windows without having to manually refresh the page?

**Comment by joshhanley** [comment]
@lasseeee it won't update other browser windows. 

You will need to use `wire:poll` https://laravel-livewire.com/docs/2.x/polling 

Or use websockets https://laravel-livewire.com/docs/2.x/laravel-echo

Hope this helps!

**Reply by kuyawing** [reply]
> @lasseeee it won't update other browser windows.
> 
> You will need to use `wire:poll` https://laravel-livewire.com/docs/2.x/polling
> 
> Or use websockets https://laravel-livewire.com/docs/2.x/laravel-echo
> 
> Hope this helps!

Polling is not instantaneous. I believe the page will refresh in 2 seconds. Can it be lowered? 

Websockets is real time but I have problems with livewire listeners

**Reply by joshhanley** [reply]
@kuyawing have a look at the poll docs page, you can add a time modifier to poll to change it to suit your needs 🙂
